index.bs

```diff
@@ -3722,6 +4314,31 @@ enum IDBCursorDirection {
 };
 </pre>
 
+
+<div class=note>
+  <dl class=domintro>
+    <dt><var>cursor</var> . {{IDBCursor/source}}</dt>
+    <dd>
+      Returns the {{IDBObjectStore}} or {{IDBIndex}} the cursor was opened from.
+    </dd>
+    <dt><var>range</var> . {{IDBCursor/direction}}</dt>
+    <dd>
+      Returns the [=cursor/direction=] of the cursor.
```

Comment:
Maybe parenthetically duplicating the contents of the enum might be helpful here (i.e. "next", "nextunique", ...)